src/recensio/plone/browser/templates/homepage.pt

```diff
@@ -26,148 +33,147 @@
-                         src="${here/portal_url}/++resource++recensio.theme.images/${langres/language}.png"
-                    />
-                  </a>
-                  <tal:res repeat="res langres/results">
-                    <li tal:define="
-                          is_first python:repeat['res'].start and 'is_first' or 'not_first';
-                        "
-                        tal:attributes="
-                          class string:${is_first} ${res/meta_type};
-                        "
-                    >
-                      <span tal:content="res/authors"></span>
-                      <span>
-                        <a class="ellipsis_container"
-                           href="#"
-                           tal:attributes="
-                             href python: request.physicalPathToURL(res['path']);
-                             class string:ellipsis_container;
-                           "
-                        >
-                          <span class="ellipsis_text"
-                                tal:content="res/title"
+                  <tal:i18n i18n:translate="label_latest_reviews">
+                  Neueste Rezensionen online
+                  </tal:i18n>
+                </a>
+              </div>
+              <div class="card-body"
+                   id="latest_reviews_box"
+                   tal:define="
+                     results view/getReviewMonographs;
+                   "
+              >
+                <span tal:repeat="langres results">
+                  <!-- <label><span tal:replace="langres/langname" /></label> -->
+                  <ul class="list_reviews"
+                      id="list_reviews"
+                      tal:attributes="
+                        id string:list_reviews_${langres/language}_${langres/part};
+                      "
+                  >
+                    <a href="${here/portal_url}/@@search?${langres/query_str}">
+                      <img alt="${langres/language}"
+                           src="${here/portal_url}/++resource++recensio.theme.images/${langres/language}.png"
+                      />
+                    </a>
+                    <tal:res repeat="res langres/results">
+                      <li tal:define="
+                            is_first python:repeat['res'].start and 'is_first' or 'not_first';
+                          "
+                          tal:attributes="
+                            class string:${is_first} ${res/meta_type};
+                          "
+                      >
+                        <span tal:content="res/authors"></span>
+                        <span>
+                          <a class="ellipsis_container"
+                             href="#"
+                             tal:attributes="
+                               href python: request.physicalPathToURL(res['path']);
+                               class string:ellipsis_container;
```

Comment:
Just a note: I‌ don't understand the purpose of `string:ellibpsis_container` here and I'd use the chameleon inline expression syntax `${var}`.
But that's copy paste from old code so :+1: anyways.